client/src/partials/debtor_groups/debtor_groups.js

```diff
@@ -0,0 +1,154 @@
+angular.module('bhima.controllers')
+.controller('DebtorGroupsController', DebtorGroupsController);
+
+DebtorGroupsController.$inject = ['$http', '$translate', 'DebtorGroupService', 'SessionService', 'uuid'];
+
+/**
+ * Debtor Groups Controller
+ * This controller is about the debtor groups management module in the admin zone
+ * It's responsible for creating and updating a debtor group
+ */
+function DebtorGroupsController($http, $translate, DebtorGroup, Session, uuid) {
+  'use strict';
+
+  var vm = this;
+
+  /** breadcrumb configurations */
+  vm.bcPaths = [
+    {
+      label: $translate.instant('DEBTOR_GRP.TITLE'),
+      current: true
+    }
+  ];
+  vm.bcButtons = [
+    {
+      dataMethod : 'create',
+      color  : 'btn-default',
+      icon   : 'glyphicon glyphicon-plus-sign',
+      label  : $translate.instant('DEBTOR_GRP.NEW'),
+      action : create
+    }
+  ];
+
+  /** init variables */
+  vm.state = {};
+  vm.debtorGroup = {};
+
+  /** Expose to the view */
+  vm.update = update;
+  vm.cancel = cancel;
+  vm.submit = submit;
+  vm.refreshValidation = refreshValidation;
+
+  /** Load debtor groups */
+  debtorGroupsList()
```

Comment:
Do these all depend on each other?  Or can they be done in parallel?  If they can be done in parallel, I encourage you to write:
```js
debtorGroupList();
accountsList();
priceList();
```
instead!  It will be more performant in practice.